integration-test/1491-france-shields.py

```diff
@@ -0,0 +1,255 @@
+from . import FixtureTest
+
+
+class FrenchShieldTest(FixtureTest):
+    def test_e_road(self):
+        import dsl
+
+        z, x, y = (16, 33186, 22554)
+
+        self.generate_fixtures(
+            dsl.is_in('FR', z, x, y),
+            # https://www.openstreetmap.org/way/16108247
+            dsl.way(16108247, dsl.tile_diagonal(z, x, y), {
+                'horse': 'no', 'hazmat': 'no', 'bicycle': 'no',
+                'name': u'Boulevard P\xe9riph\xe9rique Int\xe9rieur',
```

Comment:
I may be wrong, but we normally get these as utf-8 encoded strings right? Might be more representative if these were all utf-8 as input since that's what's going to happen in practice? eg `u'Boul...'.encode('utf-8')`

(As an aside, I think it would be much better if immediately after reading the data from the input source all strings were forced to unicode, but that's a separate bit of cleanup work)

Reply:
Yup, good point. I've changed the whole file encoding in 4197bc1 - is that closer to what you were thinking?

Reply:
That still keeps the input that the code sees as unicode though, and not utf-8 right? I was coming at it more that in practice the code will always see utf-8 strings and not unicode. ie as a pedant :P 

My understanding is that currently the test case is: `isinstance(name, str) == False and isinstance(name, unicode) == True`. Whereas the input tag that the transforms see when running in production will always be: `isinstance(name, str) == True`.

That make sense?

Reply:
Ah! Sorry, yes, I think I understand now.

I think it's easier to read to keep the `unicode` in the test file (especially since 4197bc1 changed it to `u'Boulevard Périphérique Intérieur'` without all the `\xe9`s). So in 8e45cf6, I made it so that the `dsl` functions convert `unicode` to UTF-8 encoded `str`. Hopefully this will replicate the real inputs.

What do you think?

Reply:
You could always just do: `u'Boulevard Périphérique Intérieur'.encode('utf-8')`, but what you did is probably better to ensure it's all utf-8. By the way, [we have a little function](https://github.com/tilezen/tilequeue/blob/4062c1a579bdf965a4ebc711f238d8b37de8b926/tilequeue/utils.py#L57-L74) that converts to utf-8, but it's a little heavier than what's needed here and doesn't fit exactly, so what you have already looks fine to me.

👍